news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: osu!mania 4k World Cup 2024 Mapper/Playtester Applications
+date: 2024-02-27 00:00:00 +0000
+---
+
+Are you interested in helping with the mappooling process for our very own osu!mania 4K World Cup? You're in luck!
+
+![](/wiki/shared/news/banners/OWC_Master_Logo.png)
+
+As with every year, the folks over at the 4K MWC mappooling team have opened applications for both mappers and playtesters, but this time with a multi-step process, aiming to give newcomers a chance to acquire staffing experience.
+
+### Role overview
+
+Mappers will be tasked with creating beatmaps based on a specification given by each mode's mappooling team. Mappers will be expected to incorporate feedback from the mappoolers and/or playtesters in an iterative process to refine the map until it is in a shippable state.
+
+Playtesters will be responsible for playing and giving feedback for a plethora of beatmaps (as requested by the mappooling team) to gauge aspects such as difficulty, suitability for a certain slot, and/or skill representativity among others (specific workflows vary for each team). Playtesters are generally expected to have considerable skill on their respective game mode.
+
+Please note that if you are accepted into either team, you will not be eligible to participate as a player in the tournament.
+
+### Recruitment process
+
+![](/wiki/shared/news/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications/mwc4k2024-apps-flowchart.png)
+
+All prospective helpers will have to answer the first with form, which contains questions aimed at gauging whether or not they're a good fit for the current team.
+
+Participants that clear the first stage will have to answer another form, this time with even more specific questions meant to gauge their experience, skill, and methodology on various aspects of mapping and/or playtesting. After this stage, one of four things may happen:
+
+  - The candidate is invited to staff for MWC 4K
+  - The candidate is invited to an individual interview before proceeding
+  - The candidate is invited to staff for RAFT
+  - The candidate is not invited to either team
+
+Please note that not all participants will receive invitations to the second and/or third stages, and may not receive invites even if they go through all steps. 
+
+If you are considering applying but do not feel like you have the necessary experience, we urge you to apply – the 4K MWC mappooling team alongside the osu!mania Tourney Central Council has developed a controlled-environment staff mentorship program under the Rapid Fire Tournament Series project (RAFT), which will give mappers and playtesters with less tournament staffing experience but high potential and interest an opportunity to demonstrate their ability before being fully accepted into the 4K MWC mappooling team.
```

Comment:
link RAFT

Reply:
RAFT is an invite-only thing, the form goes more in-depth but it's still a trial

Reply:
is there no information about it online?

Reply:
what about the tourney central council?

Reply:
> is there no information about it online?

It's something they created for these apps in specific, so not really

Reply:
link Tourney Central Council added on 220bacd